custom/icds_reports/tests/agg_tests/fixtures/child_cases.csv

```diff
@@ -1304,3 +1304,7 @@ test_child_case_2,2017-07-01,Name 1508,2017-06-23,2017-05-02,,0,d5d0fce5e73ff2b0
 test_child_case_3,2017-07-01,Name 1508,2017-06-23,2017-05-02,,0,d5d0fce5e73ff2b04417f40bd2b7b165,test_child_case_3,ab2ab93e-d927-421f-9d39-b8183ab96aac,new_person_case3,a103,s22,b7,d6,st5,2013-05-02,F,,1,,0,0,,0,0,0,0,0,0,0,0,0,0,0,1,,,0,yes,no,st,no,Name 1508,सिमाबाई,,,,0,1,
 test_child_case_4,2017-07-01,Name 1508,2017-06-23,2017-05-02,,0,d5d0fce5e73ff2b04417f40bd2b7b165,test_child_case_4,ab2ab93e-d927-421f-9d39-b8183ab96aac,new_person_case4,a104,s23,b8,d7,st6,2013-05-02,F,,1,,0,0,,0,0,0,0,0,0,0,0,0,0,0,1,,,0,yes,no,st,no,Name 1508,सिमाबाई,,,,0,1,
 test_child_case_5,2017-07-01,Name 1508,2017-06-23,2017-05-02,,0,d5d0fce5e73ff2b04417f40bd2b7b165,test_child_case_5,ab2ab93e-d927-421f-9d39-b8183ab96aac,new_person_case5,a105,s24,b9,d8,st7,2013-05-02,F,,1,,0,0,,0,0,0,0,0,0,0,0,0,0,0,1,,,0,yes,no,st,no,Name 1508,सिमाबाई,,,,0,1,
+test_child_case_6,2017-07-01,Name 1388,2017-06-27,2016-12-22,,0,d5d0fce5e73ff2b04417f40bd2bf3c16,test_child_case_6,e8f03ae5-820c-4f50-9e9e-5908abf92878,5ad6abf1-7781-44f9-a32a-5bc22f0a6bad,a7,s7,b4,d3,st2,2013-06-06,M,yellow,1,,0,0,2017-06-17,1,0,0,0,0,0,0,0,0,1,0,0,,,0,yes,no,sc,no,Name 1388,रेरवा,,,,0,0,
```

Comment:
were all the existing cases  are valid that you needed to add more

Reply:
no i thought they are being used in other tests. So added a few to cover more cases